app/presenters/publishing_api/editionable_worldwide_organisation_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module PublishingApi
+  class EditionableWorldwideOrganisationPresenter
+    include Rails.application.routes.url_helpers
+    include ActionView::Helpers::UrlHelper
+
+    attr_accessor :item, :update_type, :state
+
+    def initialize(item, update_type: nil, state: "published")
+      self.item = item
+      self.update_type = update_type || "major"
+      self.state = state
+    end
+
+    delegate :content_id, to: :item
+
+    def content
+      content = BaseItemPresenter.new(
+        item,
+        update_type:,
+      ).base_attributes
+
+      content.merge!(
+        details: {
+          logo: {
+            crest: "single-identity",
+          },
+        },
+        document_type: item.class.name.underscore,
+        public_updated_at: item.updated_at,
+        rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
+        schema_name: "worldwide_organisation",
+      )
+      content.merge!(PayloadBuilder::PolymorphicPath.for(item))
+    end
+
+    def links
+      {}
```

Comment:
We already have the organisations linked to the model [here](https://github.com/alphagov/whitehall/blob/b814a4ebd2600285ddd226ca5a135e156638199a/app/models/editionable_worldwide_organisation.rb#L2).  Maybe the organisations can already be included in the links?

Reply:
Only `sponsoring_organisations` are included in the links for the original presenter, given that association doesn't exist yet that probably needs to wait.

Reply:
The wording of "Sponsoring Organisations" in the UI has changed to just be "Organisations" in the editionable workflow, which we already have as an association on the model.  So you should be able to include these as the `sponsoring_organisations`, e.g.

```ruby
    def links
      {
        sponsoring_organisations: item.organisations.map(&:content_id)
      }
    end
```

Plus a corresponding test.